bp_common/syn/Makefile.vcs

```diff
@@ -40,8 +40,20 @@ VCS_BUILD_OPTS += +plusarg_save
 VCS_BUILD_OPTS += $(BP_TOOLS_DIR)/install/lib/libdromajo_cosim.a
 VCS_BUILD_OPTS += -top testbench
 VCS_BUILD_OPTS += -f flist.vcs
-
-VCS_PLUSARGS  =
+ifdef UVM
+VCS_BUILD_OPTS += -ntb_opts uvm-1.2 +incdir+$(BP_UVM_DIR)
+VCS_BUILD_OPTS += -v2k_generate
```

Comment:
The extra VCS flags should be set in test/tb/uvm/Makefile.vcs

Reply:
Looking at bp_common/syn/Makefile.vcs it includes test/tb/uvm/Makefile.vcs on line 17, but then line 27 overwrites VCS_BUILD_OPTS as just -full64.  This means that setting VCS_BUILD_OPTS in test/tb/uvm/Makefile.vcs gets overwritten before compilation, is there a way to avoid this?

Reply:
Oh, should just have VCS_BUILD_OPTS += -full64 to avoid

Reply:
For both VCS_BUILD_OPTS and VCS_PLUSARGS the = has been changed to +=.  Let me know if this is acceptable, I would not want to break other parts of BP.